packages/wallet/src/redux/channel-state/reducer.ts

```diff
@@ -111,10 +118,10 @@ const initializedChannels: ReducerWithSideEffects<states.InitializedChannelState
     action,
   );
 
-  return { state: { ...state, [appChannelId]: newState }, sideEffects: outboxState };
+  return { state: { ...state, [channelId]: newState }, sideEffects: outboxState };
 };
 
-const initializedChannelStatusReducer: ReducerWithSideEffects<states.ChannelStatus> = (
+export const initializedChannelStatusReducer: ReducerWithSideEffects<states.ChannelStatus> = (
```

Comment:
This is a bit ugly but exporting this allows us to mock it out.
It makes the routing tests a lot less fragile since they don't have to rely on logic of the channel stage reducers. 

Perhaps these should just be separated out into their own file?